### ISSUE TITLE
Rephrase "Before you signup..." message

### ISSUE THREAD

**Issue by tiagoalvesdulce** [decred/politeiagui#608]
We should rephrase it to be more concise. I think the first paragraph can be removed.

<img width="340" alt="screen shot 2018-09-18 at 13 35 31" src="https://user-images.githubusercontent.com/13955303/45702281-ca8b5080-bb47-11e8-8d41-e245013a8e6a.png">

**Comment by s-ben** [comment]
Working on this now...

**Comment by camus-code** [comment]
Think you meant to post the comment below on this issue. Reposting to keep things tidy.

"Politeia will send you a link to verify your email address. You must open this link in the same browser. After verifying your email, Politeia will create your Politeia “identity”, which consists of a public/private cryptographic key pair and browser cookie. This is necessary to verify your identity and allow submission of proposals, commenting, voting, and other Politeia functions. After completing the signup process, you can export your identity (public/private keys) to another browser at any time."

**Comment by camus-code** [comment]
@s-ben

**Comment by s-ben** [comment]
Oops. Brain fart. Good catch @camus-code

**Comment by tiagoalvesdulce** [comment]
Looks good @s-ben can you open a PR for that?

**Comment by s-ben** [comment]
Np, working on it @tiagoalvesdulce
